CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+## 16.7.0
+
+- Changed: `tap` formatter to support TAP14 ([#7759](https://github.com/stylelint/stylelint/pull/7759)) ([@Mouvedia](https://github.com/Mouvedia)).
+- Added: `url` secondary option ([#7743](https://github.com/stylelint/stylelint/pull/7743)) ([@emmacharp](https://github.com/emmacharp)).
+- Added: `fix` callback to the object passed to `stylelint.utils.report()` ([#7730](https://github.com/stylelint/stylelint/pull/7730)) ([@Mouvedia](https://github.com/Mouvedia)).
```

Comment:
The `fix` callback should not be exposed yet.

```suggestion
```

See:
https://github.com/stylelint/stylelint/blob/bb72fc7937d74a2579622d23a611a20c5276b4ba/types/stylelint/index.d.ts#L633-L637